src/components/edit_alarm_definition.js

```diff
@@ -22,10 +22,10 @@ import _ from 'lodash';
 export class EditAlarmDefinitionPageCtrl {
 
   /** @ngInject */
-  constructor($scope, $injector, $location, backendSrv, datasourceSrv, alertSrv) {
+  constructor($scope, $injector, $location, backendSrv, datasourceSrv, alertSrv, monascaClientSrv) {
```

Comment:
Can we remove backendSrv, datasourceSrv now that you're injecting the monasca client? They don't appear to be used?

Reply:
+1